javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,15 @@
+/* exported filterOutNulls */
+
+function filterOutNulls(values){
+  var onlyNums = [];
+
+  for (var i = 0; i < values.length; i++){
+    if (typeof values[i] !== 'object'){
+      onlyNums.push(values[i]);
+    }
+    else{
```

Comment:
This block is unnecessary. If the conditional doesn't run, your code will continue on its own 👍